

17:56:26 error: Querying against a released or lost connection.
         task(user-check): insert into "user_edu_person_entitlement"("user_id","edu_person_entitlement") values(2,'urn:mace:egi.eu:group:service-integration.aai.egi.eu:role=member#aai.egi.eu'),(2,'urn:mace:egi.eu:group:service-integration.aai.egi.eu:role=vm_operator#aai.egi.eu')